src/test/java/RegistrationTest.java

```diff
@@ -0,0 +1,42 @@
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.chrome.ChromeDriver;
+import org.testng.annotations.*;
+import pages.RegistrationPage;
+
+import static org.testng.Assert.assertTrue;
+
+public class RegistrationTest {
+    WebDriver driver;
+    RegistrationPage registrationPage;
+
+    @BeforeClass
+    public void setup() {
+        driver = new ChromeDriver();
+        driver.manage().window().maximize();
+        driver.get("https://percab.uzairways.com/auth/register");
+        registrationPage = new RegistrationPage(driver);
+    }
+
+    @Test
+    public void testUserRegistration() throws InterruptedException {
+        registrationPage.enterEmail("testuser@example.com");
+        registrationPage.enterFirstName("Aziza");
+        registrationPage.enterLastName("Makhmudova");
+        registrationPage.enterBirthDate("01.01.2000");
+        registrationPage.enterPassword("Password123");
+        registrationPage.confirmPassword("Password123");
+
+        System.out.println("Please solve the CAPTCHA manually in the browser");
```

Comment:
Please remove sout. It's not necessary.

A better way to print out info is logging, we will add that later as it's separate logic

Reply:
Also we CANNOT ask anyone for manual actions in the test. The test should be fully automated 

In case of captcha - it should fail, as according to the requirments from product owner (me) - captcha is a bug